src/animation/AnimatedComponent.js

```diff
@@ -19,14 +20,15 @@ export default class AnimatedComponent extends PureComponent<Props> {
   static defaultProps = {
     visible: true,
     useNativeDriver: true,
+    duration: 300,
   };
 
   animatedValue = new Animated.Value(0);
 
   componentWillReceiveProps(nextProps: Props) {
     Animated.timing(this.animatedValue, {
       toValue: nextProps.visible ? nextProps[nextProps.property] : 0,
-      duration: 300,
```

Comment:
I definitely want to keep these animated components less configurable so it is more consistent (and simpler).

Reply:
Do you really need this to be 600msecs vs 300msecs?

Reply:
Reverted back